site/ko/beta/guide/saved_model.ipynb

```diff
@@ -0,0 +1,1176 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "9WjGEaS3XfX7"
+   },
+   "outputs": [],
+   "source": [
+    "labeling = infer(tf.constant(x))[pretrained_model.output_names[0]]\n",
+    "\n",
+    "decoded = imagenet_labels[np.argsort(labeling)[0,::-1][:5]+1]\n",
+    "\n",
+    "print(\"저장과 불러오기 이후의 결과:\\n\", decoded)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "SJEkdXjTWbtl"
+   },
+   "source": [
+    "## 모델 배포하기\n",
+    "\n",
+    "SavedModels는 파이썬에서 사용하기에 적합하지만, 일반적으로 프로덕션 환경에서는 추측 전용 서비스를 사용합니다. 이는 TensorFlow Serving을 사용한 SavedModel에서 쉽게 설정할 수 있습니다.\n",
+    "\n",
+    "`tensorflow_model_server`를 노트북이나 로컬 머신에 설치하는 방법을 포함한 배포에 대한 자세한 내용은 [TensorFlow Serving REST 튜토리얼] (https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/tutorials/Serving_REST_simple.ipynb)을 참조하십시오. 임시방편으로 앞서 내보낸 `mobilenet` 모델을 배포하기 위해 모델 서버를 SavedModel 디렉토리로 설정합니다:\n",
+    "\n",
+    "```bash\n",
+    "nohup tensorflow_model_server \\\n",
+    "  --rest_api_port=8501 \\\n",
+    "  --model_name=mobilenet \\\n",
+    "  --model_base_path=\"/tmp/mobilenet\" >server.log 2>&1\n",
+    "```\n",
+    "\n",
+    "  이제 요청을 보냅니다.\n",
+    "\n",
+    "```python\n",
+    "!pip install requests\n",
+    "import json\n",
+    "import numpy\n",
+    "import requests\n",
+    "data = json.dumps({\"signature_name\": \"serving_default\",\n",
+    "                   \"instances\": x.tolist()})\n",
+    "headers = {\"content-type\": \"application/json\"}\n",
+    "json_response = requests.post('http://localhost:8501/v1/models/mobilenet:predict',\n",
+    "                              data=data, headers=headers)\n",
+    "predictions = numpy.array(json.loads(json_response.text)[\"predictions\"])\n",
+    "```\n",
+    "\n",
+    "`predictions`의 결과는 파이썬에서의 결과와 동일합니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Bi0ILzu1XdWw"
+   },
+   "source": [
+    "### SavedModel 포맷\n",
+    "\n",
+    "SavedModel은 연속된 시그니처와 이를 실행하는 데 필요한 변수값과 어휘를 포함한 상태를 포함하는 디렉토리입니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "6u3YZuYZXyTO"
+   },
+   "outputs": [],
+   "source": [
+    "!ls /tmp/mobilenet/1  # assets\tsaved_model.pb\tvariables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ple4X5utX8ue"
+   },
+   "source": [
+    "`saved_model.pb` 파일은 함수를 식별하는 각각의 알려진 시그니처 세트를 포함합니다.\n",
+    "\n",
+    "SavedModels에는 다중 시그니처 세트(`tag_set`인수가 `saved_model_cli`로 확인된 다중 MetaGraph)를 포함할 수 있지만 이런 경우는 드뭅니다. 다중 시그니처 세트를 작성하는 API에는 [`tf.Estimator.experimental_export_all_saved_models`](https://www.tensorflow.org/api_docs/python/tf/estimator/Estimator#experimental_export_all_saved_models) 및 TensorFlow 1.x의 `tf.saved_model.Builder`가 포함됩니다."
```

Comment:
We translate argument as '매개변수 값', because many people use '인자' and '매개변수' interchangeable.
I suggest "\`tag_set\`인수가 \`saved_model_cli\`로" --> "\`saved_model_cli\`의 \`tag_set\` 매개변수 값으로"